### ISSUE TITLE
useTransition: items don't unmount after leaving

### ISSUE THREAD

**Issue by desmap** [react-spring/react-spring#422]
I added this https://github.com/react-spring/react-spring/issues/377#issuecomment-451639427 to an closed issue, not sure if it got your attention. So, I am opening a new issue, hope that's ok. The issue:

_this bug seems to be back, check https://codesandbox.io/s/98j711344o and observe class='bg' in the Inspector. It doesn't get unmounted (react-spring v7.2.7)._

**Comment by drcmda** [comment]
@ruggedy you know what it could be?

**Comment by drcmda** [comment]
weird, it doesn't do this locally, i think i'll publish a new build today and hopefully this goes away

**Comment by drcmda** [comment]
ok, solved, in part, it does unmount, but only after all springs have come to rest. our local demo had a delay of 3000ms. this isn't correct though, it should unmount immediately. still looking into this ...

**Comment by drcmda** [comment]
fixed - will release later this day, there's some other issue that needs to be fixed

**Comment by desmap** [comment]
Awesome! I just tested it on react-spring@7.2.8-beta.1. It works like a charm. 🥇
